xinference/model/llm/llm_family.py

```diff
@@ -380,20 +380,26 @@ def _get_meta_path(
     cache_dir: str,
     model_format: str,
     model_hub: str,
+    model_revision: Optional[str],
     quantization: Optional[str] = None,
 ):
     if model_format == "pytorch":
         if model_hub == "huggingface":
-            return os.path.join(cache_dir, "__valid_download")
+            return os.path.join(cache_dir, f"__valid_download_{model_revision}")
```

Comment:
The meta file has included the model revision , see`_generate_meta_file`. Is it necessary to add model revision at the end of the filename?

Reply:
> The meta file has included the model revision , see`_generate_meta_file`. Is it necessary to add model revision at the end of the filename?

But you are deciding whether to download the model or not according to the file `__valid_download`.